examples/inverse/plot_mce_inverse.py

```diff
@@ -0,0 +1,70 @@
+"""
+=======================================================
+Compute minimum current estimate (MCE) inverse solution
+=======================================================
+
+Runs a minimum-current estimate [1]_ by using the
+:func:`mne.inverse_sparse.mixed_norm` function [2]_.
+L0.5/L2 is done with irMxNE which allows for sparser
+source estimates with less amplitude bias due to the non-convexity
+of the L0.5/L2 mixed norm penalty.
+
+References
+----------
+.. [1] Uutela K, Hamalainen M, and Somersalo E. 1999.
+   "Visualization of magnetoencephalographic data using minimum current
+   estimates". Neuroimage, 10:173-80
+.. [2] Gramfort A., Kowalski M. and Hamalainen, M.
+   "Mixed-norm estimates for the M/EEG inverse problem using accelerated
+   gradient methods", Physics in Medicine and Biology, 2012.
+   http://dx.doi.org/10.1088/0031-9155/57/7/1937
+"""
+# Authors: Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#          Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import mne
+from mne.datasets import sample
+from mne.inverse_sparse import mixed_norm
+from mne.viz import plot_sparse_source_estimates
+
+print(__doc__)
+
+data_path = sample.data_path()
+fwd_fname = data_path + '/MEG/sample/sample_audvis-meg-eeg-oct-6-fwd.fif'
+ave_fname = data_path + '/MEG/sample/sample_audvis-ave.fif'
+cov_fname = data_path + '/MEG/sample/sample_audvis-shrunk-cov.fif'
+subjects_dir = data_path + '/subjects'
+
+# Read noise covariance matrix
+cov = mne.read_cov(cov_fname)
+# Handling average file
+condition = 'Left Auditory'
+# Let's crop to a time of interest
+evoked = mne.read_evokeds(ave_fname, condition=condition, baseline=(None, 0))
+# Handling forward solution
+forward = mne.read_forward_solution(fwd_fname, surf_ori=True)
+
+ylim = dict(eeg=[-10, 10], grad=[-400, 400], mag=[-600, 600])
+evoked.plot(ylim=ylim, proj=True)
+
+###############################################################################
+# Run solver
+
+alpha = 50  # regularization parameter between 0 and 100 (100 is high)
+loose, depth = 0.2, 0.9  # loose orientation & depth weighting
+
+# Compute MCE: equivalent to unweighted MxNE on a single time point!
+# To solve MCE for multiple time points, you could loop through them
+# one-by-one since they are treated independently.
```

Comment:
We should point out that computing MCE for multiple time points by running MxNE for each sample seperately applies a different regularization parameter per time point. Even though alpha is the same, alpha_max varies between samples and the resulting regularization parameter lambda = alpha / 100 * alpha_max  is different. The alternative is to have an MCE function in the sparse module, which takes a time series, computes alpha_max by taking the max over samples, and then either loops over samples with MxNE to compute the MCE estimate per sample or applies MCE solver, which is, however, not yet part of MNE (I think the first approach should be good enough)